<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administrateur_assistant_page</class>
 <widget class="QDialog" name="administrateur_assistant_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart cinema</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/new/prefix3/beige.jpg);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>661</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix2/marron.jpg);</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Administrateur</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>331</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajout</string>
     </property>
     <widget class="QPushButton" name="ajouter_administrateur">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifieradmin">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomadmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenomadmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailadmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idadmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'utilisateur :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameadmin">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordadmin">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>291</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Affichage</string>
     </property>
     <widget class="QTableView" name="tableViewadmin">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>241</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="afficheradmin">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>192</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimeradmin">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>192</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>191</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>suppression</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idsuppadmin">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimeradmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>92</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;
</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="cinema">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/a.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>461</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Email</string>
     </property>
     <widget class="QLineEdit" name="emailenvoie">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sujetenvoie">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Sujet</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Message :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="envoyer">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>81</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="messagenvoyer">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>350</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: whit</string>
     </property>
    </widget>
    <zorder>cinema</zorder>
    <zorder>groupBox</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>groupBox_8</zorder>
    <zorder>messagenvoyer</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Assistant</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>281</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Affichage</string>
     </property>
     <widget class="QTableView" name="tableViewassis">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>241</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="afficherassis">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;
</string>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>311</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajout</string>
     </property>
     <widget class="QPushButton" name="ajouterassis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:1px;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifierasis">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomassis">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenomassis">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailassis">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idassis">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NOMBREHASSIS">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="nbrh">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de travai :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordassis">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'utilisateur :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernamassis">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>201</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Suppression</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="suppassisid">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimerassis">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>280</y>
       <width>271</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Recherche</string>
     </property>
     <widget class="QPushButton" name="rechercherassistant">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idrecherche">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewrechercher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>256</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>24</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID :</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
